docs/references/namespaces.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_label: "Namespaces"
+title: ""
+---
+
+# Namespaces
+
+Epinio has the same concept of namespaces as the underlying Kubernetes, i.e.
+a means of grouping applications, configurations, and services into some kind
+of related sets.
+
+In contrast to Kubernetes the Epinio client however maintains local state, the
+__current namespace__, often also called the __targeted namespace__.
+
+Where users of Kubernetes's client (`kubectl`) have to explicitly specify the
+namespace to work with with each command, users of Epinio's client (`epinio`)
+have one command to set and query the current namespace, and all other commands
+of the client use that namespace in their operation.
+
+The relevant command is [epinio target](commands/cli/epinio_target.md).
```

Comment:
Perhaps mention on how to use the `epinio namespace` in the command reference? 
https://github.com/epinio/docs/blob/24c80338d840a9f5ab8979d7c7aa979f72878040/docs/references/commands/cli/namespace/epinio_namespace.md

Reply:
Yes, definitely.

Reply:
Resolved with commit d0e745295ac49d97abc5d5b55d2c1495d7efb574